src/data/clients.ts

```diff
@@ -128,8 +105,8 @@ export const Clients: Array<Client> = [
     platforms: [Platform.Browser],
     primaryLinks: [
       {
-        id: 'browser',
-        name: 'Open in Browser',
+        id: 'website',
+        name: 'Website',
```

Comment:
We differentiate between "hosted" clients and the actual websites advertising the client.

you've now renamed all of those to "website" but this is a hosted instance, I don't think that's a good change. (same for preserve/feishin)